week6/Homework6-start/Homework6/View/TaskRowView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  TaskRow.swift
+//  
+
+import SwiftUI
+
+struct TaskRowView: View {
+    @ObservedObject var taskStore: TaskStore
+    @State var taskStatus: Bool = false
+    
+  let task: Task
+  var body: some View {
+    return HStack {
+      Text(task.title)
+      Spacer()
+      Image(systemName: taskStatus ? "checkmark.square" : "square")
+        .foregroundColor(taskStatus ? Color.green : Color.red)
+        .animation(.default, value: taskStatus)
+        .onTapGesture {
+            taskStatus.toggle()
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```

Comment:
You got it! Perfect implementation to allow for the animation to show before the list row disappears. Buuuuut since I know you are a leet dev, I challenge you to create a more complex animation than using the default one.